connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -139,7 +141,8 @@ public class ErrorHandlingTaskTest {
     @SuppressWarnings("unused")
     @Mock
     private SourceTask sourceTask;
-    private Capture<WorkerSinkTaskContext> sinkTaskContext = EasyMock.newCapture();
+    @Captor
+    private ArgumentCaptor<WorkerSinkTaskContext> sinkTaskContext;
```

Comment:
I think that this argument capture isn't being used anywhere. We never call `sinkTaskContext.getValue()`. Perhaps we can remove this?